torch/csrc/autograd/record_function.cpp

```diff
@@ -197,6 +197,7 @@ void RecordFunction::processCallbacks() {
       manager().start_callbacks[idx](*this);
     }
   }
+  state |= BEFORE_CALLBACKS_RAN;
```

Comment:
What does `BEFORE_CALLBACKS_RAN` mean here? Has `manager().start_callbacks[idx](*this);` already started the callbacks?

Reply:
I guess the name here is a little confusing it means "before callbacks have run", maybe renaming it to "RAN_BEFORE_CALLBACKS" might be clear?

Reply:
Yes, I will rename this.